src/reducers/services.js

```diff
@@ -35,6 +36,9 @@ function fetchStarted(state) {
 
 function fetchServicesDone(state, { payload }) {
   const services = payload.data;
+  if (Array.isArray(services)) {
+    services.sort(baseStringComparator);
```

Comment:
In my opinion, I think we should use lodash as much as we can on util stuff. 

It's well tested, easier to read and immutable.
```js
const sortedServices = _.sortBy(services, s => s)
// or if it was an obj { "name": "svc1" }
const sortedServices = _.sortBy(services, 'name')

```

Reply:
@saminzadeh I would generally go with `arr.sort()` instead of `_.sortBy(arr, value => value)`. You prefer the lodash util in this case?

The sort util function is using [`string#localCompare(...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) with the option [`sensitivity`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#Using_options). It's ignores accents, case, etc.

I don't think lodash gets us anywhere, here, because it takes a transform function instead of a comparator. 

Can you elaborate?

Reply:
Other than keeping things immutable (redux approach), I think its just about keeping it consistent. Both work, just my 2 cents.